PyPDF2/generic/_annotations.py

```diff
@@ -165,6 +165,33 @@ def line(
         )
         return line_obj
 
+    @staticmethod
+    def square(
```

Comment:
Naming a method that can draw any sized rectangle "square" seems unnecessarily confusing?

Not all rectangles are squares, but all squares are rectangles and all that.

Reply:
I agree however the standard calls that Square...

Reply:
Yes, I'm also not happy about that. I chose it because of the PDF specs ... but it might make more sense to call it "rectangle" instead :thinking:

Reply:
I did note it in the docs, but it might be more reasonable to note that the rectangle method makes use of the "square" PDF annotation

Reply:
@pubpub-zz @MasterOdin I've renamed the `square` method to `rectangle`.

Reply:
Yeah, I think this is a "bug" in the spec, and that can be seen in many tools having the tool to do this called "rectangle" (and use a rectangle icon?) vs calling it "square".

I do think this is an example of are the end users of PyPDF2 super knowledgable in the spec and so would think of "square" vs the layperson who'd probably think "rectangle". One thing would be to have `AnnotationBuilder.square` be an alias of `AnnotationBuilder.rectangle` (or vice versa) and have the former just internally call and return the latter, so maintenance overhead would be minimal. I'd wait till someone complains about wanting something like that first though.

Reply:
I agree that the specs should have called it differently :+1: 

I also think that we write PyPDF2 towards developers who are not necessarily knowledgable in the PDF specs (I'm not sure if you meant it like that).

Another example is the `.metadata` attribute which I favored instead of `.info` or `document_info`. Or in general that we use snake_case attributes now instead of CamelCase.

In general, I think we do a good job if more and more people start using PyPDF2 and the amount of questions gets less / more specific. And I can already smell people asking "how do I make a rectangle" if we only add a "square" annotation :smile:

Reply:
> I also think that we write PyPDF2 towards developers who are not necessarily knowledgable in the PDF specs (I'm not sure if you meant it like that).

Yeah, and so these people would probably look for "rectangle" as that's what they want and what it's called, and they don't know the spec disagrees with them.